### DISCUSSION TITLE
Time Quantum feature for task Scheduling

### DISCUSSION THREAD

**Discussion by yoga95lohumi** [manoelcampos/cloudsim-plus#417] [Q&A] [answered]
is there a way to implement time slicing in cloudSim plus ?
Lets say I need to implement Round Robin Scheduling with dynamic Time quantum , is it poosible ?

**Comment by manoelcampos** [comment, marked as answer]
Check https://github.com/cloudsimplus/cloudsimplus-examples/blob/master/src/main/java/org/cloudsimplus/examples/schedulers/LinuxCompletelyFairSchedulerExample.java
